README.md

```diff
@@ -237,6 +237,23 @@ A simple but complete example to deploy a full monitoring stack on Kubernetes ca
 
 A complete example monitoring stack implemented in Docker Compose which can be found in the [examples](examples/config/docker) directory. 
 
+#### TLS HTTPS Support
+
+An usage example of how to deploy the exporter with TLS.
```

Comment:
```suggestion
An usage example of how to deploy the exporter with TLS using the `pure-fa-om-exporter` binary.
```

Reply:
Agreed, I have submitted a new commit to cover this and another minor adjustment in commit 2945c45.